docs/docs/integrations/platforms/google.mdx

```diff
@@ -121,6 +121,67 @@ from langchain_google_vertexai import VertexAIModelGarden
 
 ## Vector Stores
 
+### AlloyDB for PostgreSQL
+
+> [AlloyDB](https://cloud.google.com/alloydb) is a fully managed relational database service that offers high performance, seamless integration, and impressive scalability on Google Cloud. AlloyDB is 100% compatible with PostgreSQL.
+
+Install the python package:
+
+```bash
+pip install langchain-google-alloydb-pg
+```
+
+See [usage example](/docs/integrations/vectorstores/alloydb).
```

Comment:
Links are equal to the file names. It should be
`(/docs/integrations/vectorstores/google_alloydb)`
The same for other links of the usage examples below.

Reply:
fixed! Thanks for catching